test/unit/http_transport_test.py

```diff
@@ -0,0 +1,368 @@
+from typing import Optional
+from unittest.mock import Mock
+
+import pytest
+from packaging.version import Version
+
+from pyexasol import (
+    ExaConnection,
+    ExaFormatter,
+)
+from pyexasol.http_transport import (
+    ExportQuery,
+    ImportQuery,
+    SqlQuery,
+)
+
+
+@pytest.fixture
+def connection():
+    mock = Mock(ExaConnection)
+    mock.options = {"encryption": True, "quote_ident": "'"}
+    mock.exasol_db_version = Version("8.32.0")
+    mock.format = ExaFormatter(connection=mock)
+    return mock
+
+
+@pytest.fixture
+def sql_query(connection):
+    return SqlQuery(connection=connection, compression=True)
+
+
+@pytest.fixture
+def import_sql_query(connection):
+    return ImportQuery(connection=connection, compression=True)
+
+
+@pytest.fixture
+def export_sql_query(connection):
+    return ExportQuery(connection=connection, compression=True)
+
+
+class TestSqlQuery:
+    @staticmethod
+    @pytest.mark.parametrize(
+        "columns,expected",
+        [(None, ""), (["LASTNAME", "FIRSTNAME"], '("LASTNAME","FIRSTNAME")')],
+    )
+    def test_build_columns_str(sql_query, columns, expected):
+        sql_query.columns = columns
+        assert sql_query._build_columns_str() == expected
+
+    @staticmethod
+    @pytest.mark.parametrize(
+        "csv_cols,expected",
+        [
+            pytest.param(None, "", id="none_specified_returns_empty_string"),
+            pytest.param(["1..2"], "(1..2)", id="values_specified_returns_value"),
+        ],
+    )
```

Comment:
Please add test cases for 
* more supported variants, e.g. "123", `FORMAT='abc'`